### PR TITLE
feature : created register form and route for signup

### PR DESCRIPTION
**Pull request by chandra1899**
issue :- #178

screenshot

image

### CODE REVIEW

**Comment by chandra1899** [comment]
can anyone review my code .............

**Review by polymahh — COMMENTED** [review]
thank you for your hard work @chandra1899 , i think we can  make SideContent in its own component since it is repeated in login page , and email input should have email icon , other than that everything is good

**Comment by chandra1899** [comment]
> thank you for your hard work @chandra1899 , i think we can make SideContent in its own component since it is repeated in login page , and email input should have email icon , other than that everything is good

hey , can u elaborate what changes i need to make in my pull request.

**Comment by chandra1899** [comment]
> @chandra1899 sorry i wasn't clear, just make the SideContent component in a new file ... /src/components/SideContent so we can import it in login and register page since it is the same in both and use MailOutlined instead of UserOutlined for the email input icon

done 👍 pull request updated

**Review by polymahh — APPROVED** [review]
tested and approved

**Comment by waynemorphic** [comment]
On registration, is there an event handler to pick the user details and store them in a database or something?

File: frontend/src/pages/Register.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import AuthLayout from '@/layout/AuthLayout';
+import SideContent from '@/components/SideContent';
+
+import logo from '@/style/images/logo.png';
+
+const { Content } = Layout;
+const { Title } = Typography;
+
+const RegisterPage = () => {
+  const onFinish = () => {};
```
Comment (waynemorphic) [inline frontend/src/pages/Register.jsx:16]: Could this event handler be implemented in such a way that values entered by the user are stored in a db or something? I can see that there are validation rules, which is great.

**Review by waynemorphic — COMMENTED** [review]
onFinish() event handler business logic

File: frontend/src/pages/Register.jsx (hunk 1 above)
Comment (waynemorphic) [reply frontend/src/pages/Register.jsx:16]: Cool, I will be watching this space 😄

Comment (waynemorphic) [reply frontend/src/pages/Register.jsx:16]: I believe [this](https://github.com/idurar/idurar-erp-crm/pull/222/files) PR is related to the registration back end business logic

**Review by onfranciis — APPROVED** [review]

**Review by Ando22 — APPROVED** [review]

**Pull request merged**